h/models/user.py

```diff
@@ -9,6 +10,10 @@
 from h.exceptions import InvalidUserId
 from h.util.user import format_userid, split_user
 
+if TYPE_CHECKING:
+    from models.group import Group  # pragma: nocover
```

Comment:
I wanted to include a general nocover for all of this in the template

https://github.com/hypothesis/cookiecutters/pull/188/files#diff-b105dedfda4c7fa56f7675db2193281714ba9835358f20e7c2562c073eb56283R167

Reply:
Oh nice, changed it to this within h directly for now